capstone/src/test/java/com/google/univiz/api/resource/MapsResourceImplTest.java

```diff
@@ -33,113 +34,111 @@
 
   @Inject private MapsResourceImpl mapsImpl;
 
-  private static final int COLLEGE_ID = 193900;
-  private static final String NAME = "New York University";
-  private static final String CITY = "New York";
-  private static final boolean FLAG_MAIN_CAMPUS = true;
-  private static final double LATITUDE = 40.729452;
-  private static final double LONGITUDE = -73.997264;
+  private static final CollegeId NYU_COLLEGE_ID = CollegeId.create(193900);
+  private static final String NYU_NAME = "New York University";
+  private static final String NYU_CITY = "New York";
+  private static final boolean NYU_FLAG_MAIN_CAMPUS = true;
+  private static final double NYU_LATITUDE = 40.729452;
+  private static final double NYU_LONGITUDE = -73.997264;
   private static final int CARNEGIE_SIZE_DEGREE = 17;
-  private static final double ADMISSION_RATE = 0.1999;
-  private static final double AVG_SAT = 1419.0;
-  private static final int NUM_OF_UNDERGRADS = 26339;
-  private static final int AVG_COST = 69830;
-  private static final double RATIO_OF_MEN = 0.4253;
-  private static final double RATIO_OF_WOMEN = 0.5747;
+  private static final double NYU_ADMISSION_RATE = 0.1999;
+  private static final double NYU_AVG_SAT = 1419.0;
+  private static final int NYU_NUM_OF_UNDERGRADS = 26339;
+  private static final int NYU_AVG_COST = 69830;
+  private static final double NYU_RATIO_OF_MEN = 0.4253;
+  private static final double NYU_RATIO_OF_WOMEN = 0.5747;
+
+  private static final CollegeId STANFORD_COLLEGE_ID = CollegeId.create(243744);
+  private static final String STANDFORD_NAME = "Stanford University";
+  private static final String STANDFORD_CITY = "Stanford";
+  private static final boolean STANFORD_FLAG_MAIN_CAMPUS = true;
+  private static final double STANFORD_LATITUDE = 37.429434;
+  private static final double STANFORD_LONGITUDE = -122.167359;
+  private static final double STANFORD_ADMISSION_RATE = 0.0436;
+  private static final double STANFORD_AVG_SAT = 1497.0;
+  private static final int STANFORD_NUM_OF_UNDERGRADS = 7083;
+  private static final int STANFORD_AVG_COST = 69109;
+  private static final double STANFORD_RATIO_OF_MEN = 0.4991;
+  private static final double STANFORD_RATIO_OF_WOMEN = 0.5009;
```

Comment:
With the exception of name, id, and latitude, you're not using these constants anywhere now. It's only used to build another constant text object. So you can inline them.